README.md

```diff
@@ -415,6 +416,18 @@ Shows Java version.
 | `SPACESHIP_JAVA_SYMBOL` | `☕` | Character to be shown before Java version |
 | `SPACESHIP_JAVA_COLOR` | `magenta` | Color of Java section |
 
+### Gradle (`gradle` )
+
+Shows Gradle version.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_GRADLE_SHOW` | `true` | Show current Gradle version or not |
+| `SPACESHIP_GRADLE_PREFIX` | `on ` | Prefix before the Gradle section |
+| `SPACESHIP_GRADLE_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after the Gradle section |
+| `SPACESHIP_GRADLE_SYMBOL` | `⬡` | Character to be shown before Gradle version |
```

Comment:
shouldn't the symbol be `🐘` since the logo has an elephant?

Reply:
@ellaqezi I original thought the same thing, but the elephant is being used by the php module.